opcua/common/ua_utils.py

```diff
@@ -256,3 +258,9 @@ def get_nodes_of_namespace(server, namespaces=[]):
     nodes = [server.get_node(nodeid) for nodeid in server.iserver.aspace.keys()
              if nodeid.NamespaceIndex != 0 and nodeid.NamespaceIndex in namespace_indexes]
     return nodes
+
+
+def register_extension_object(object_factory):
+    setattr(ObjectIds, "{}_Encoding_DefaultBinary".format(object_factory.__name__), object_factory.DEFAULT_BINARY)
+    ObjectIdNames[object_factory.DEFAULT_BINARY] = object_factory.__name__
+    ExtensionClasses[object_factory.DEFAULT_BINARY] = object_factory
```

Comment:
I just saw that it won't work if your NamespaceIndex is not 0. I'll see what I can do to change that.